pkg/apis/servicecatalog/v1alpha1/register.go

```diff
@@ -56,6 +57,16 @@ func addKnownTypes(scheme *runtime.Scheme) error {
 		&metav1.GetOptions{},
 
 		&Broker{},
+		&BrokerList{},
+		&ServiceClass{},
+		&ServiceClassList{},
+		&Instance{},
+		&InstanceList{},
+		&Binding{},
+		&BindingList{},
 	)
+
+	versionedwatch.AddToGroupVersion(scheme, SchemeGroupVersion)
```

Comment:
in the unversioned code that's similar to this we don't have this line. Is this associating our "versioned" objects with the "unversioned" ones?